outrank/algorithms/synthetic_data_generators/cc_generator.py

```diff
@@ -0,0 +1,702 @@
+import numpy as np
+from scipy.linalg import qr
+from scipy.stats import norm
+from sklearn.cluster import KMeans
+from sklearn.utils import resample
+from typing import List, Union, Optional, Tuple
+class CategoricalClassification:
```

Comment:
Did you install `pre-commit` hook? Please run it, spacing looks off

Reply:
pre-commit installed, ran